Assets/VRTK/Scripts/Presence/VRTK_BodyPhysics.cs

```diff
@@ -1035,13 +1038,15 @@ protected virtual void StartFall(GameObject targetFloor)
             isMoving = false;
             isLeaning = false;
             onGround = false;
+            fallMinTime = Time.time + (Time.fixedDeltaTime * 3.0f); // Wait at least 3 fixed update frames before declaring falling finished 
```

Comment:
Just want to be sure physics has enough time to settle down or add gravity and such. It's a buffer zone really, so any amount of time would work. What I really need is the ability to know if physics has been put to sleep. The velocity has always been a bit of a hack.